tasks/push-snapshot/push-snapshot.yaml

```diff
@@ -111,6 +119,9 @@ spec:
             --format '{{.Digest}}' \
             "docker://${containerImage}" 2>/dev/null)
 
+          RESULTS_JSON=$(jq --arg i "$i" --arg arch "$arch" --arg name "$name" --arg sha "$origin_digest" \
+            '.images[$i|tonumber] += {"arch": $arch, "name": $name, "shasum": $sha, "urls": []}' <<< "$RESULTS_JSON")
```

Comment:
Opening this thread as a reminder that I need to list all the archs here, not just the first one